DigiMeSDK/Repository/Classes/Objects/Social/Post.swift

```diff
@@ -56,20 +56,19 @@ public class Post: NSObject, BaseObjectDecodable {
     public let likeCount: Int
     public let links: [Link]?
     public let longitude: Double
-//    public let originalCrossPostIdentifier: Int
-//    public let originalPostIdentifier: Int
+    public let originalPostIdentifier: String?
     public let originalPostUrl: String
     public let personIdentifier: String
     public let personFileUrl: String
     public let personFullname: String
     public let personUsername: String
     public let postIdentifier: String
-//    public let postReplyCount: String
+    public let postReplyCount: AnyJSONType?
```

Comment:
I don't understand what `AnyJSONType` is here? Can't we just map it onto a base type? Like a `String`?

I understand it may come as an `Int`, or as a `Double` etc, but we should still be able to provide a type back to the user that they *actually* know.

Here's an example:

```swift

        if let val = anyVal as? String {
            return val
        }
        else if let val = anyVal as? Int {
            return "\(val)"
        }
        else if let val = anyVal as? Double {
            return "\(val)"
        }
```

Regardless of what the type is - it is mapped onto a String. You partially have that, so perhaps we can re-use it.

I don't think we should be "passing the multi type" problem onto the user.